src/components/Events.module.scss

```diff
@@ -1,8 +1,26 @@
-.container {
+.event {
+  align-items: left;
```

Comment:
Does this even work when this is not display flex?

Reply:
It does not. Deleted